### DISCUSSION TITLE
[월드 1.1.6] index.ts(서버 코드) onLeave 함수 호출 안됨

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#604] [5. Q&A]
월드 1.1.6 입니다 

아래는 기본 예제 코드 입니다 
![image](https://user-images.githubusercontent.com/50188795/162628118-47a334ec-52d5-4562-b7a8-5585a5a059ea.png)

월드 1.1.5에서 유저가 나갔을때 호출되었던 onLeave가 호출 안되는듯 합니다 

그리고 월드 1.1.6에서 1.1.5로 다운그레이드 해서 테스트 해보려고 했더니 
다음과 같이 WorldMultiplay 컴포넌트가 안보여요 ~ 
![image](https://user-images.githubusercontent.com/50188795/162628413-e296fff0-8221-43d2-8e72-7402d1cdebdb.png)
![image](https://user-images.githubusercontent.com/50188795/162628418-d83b1511-f479-472b-b58e-5808f8ca9b36.png)
다시 1.1.6으로 돌아와도 WorldMultiplay컴포넌트는 안보입니다.. ㅠㅠ
![image](https://user-images.githubusercontent.com/50188795/162628511-1c4a6cbd-9cd6-49e6-97ae-19a87e102414.png)
![image](https://user-images.githubusercontent.com/50188795/162628520-d82f6823-a30c-4451-aa11-d5d41d97fb5d.png)

어찌된일일까요....~.~

확인좀 부탁 해요

**Comment by pumfit** [comment]
world 1.1.6 패키지 업데이트 한 뒤에 상단 Assets - Reimport All을 진행하면 정상적으로 WorldMultiplay컴포넌트가 나타납니다!
